docs/scripts_md/get_dicom_files.md

```diff
@@ -4,7 +4,7 @@ get\_dicom\_files.pl - extracts DICOM files for specific patient names/scan type
 
 # SYNOPSIS
 
-perl get\_dicom\_files.pl \[-names patient\_name\_patterns\] \[-types scan\_type\_patterns\] \[-outdir tmp\_dir\] \[-outfile tarBasename\] 
+perl get\_dicom\_files.pl \[-name patient\_name\_patterns\] \[-type scan\_type\_patterns\] \[-outdir tmp\_dir\] \[-outfile tarBasename\] 
```

Comment:
The usage is missing options `-name`, `-dumpOptions`, `-nofilesbk` and `-nosqlbk`. Don't forget to regenerate the .`md` files when you correct this!   :)

Reply:
this is the MD file for `get_dicom_files.pl`. I regenerated all the MD files of the PR.